client/src/modules/enterprises/enterprises.js

```diff
@@ -106,6 +106,10 @@ function EnterpriseController(Enterprises, util, Notify, Projects, Modal, Scroll
     const creation = (vm.hasEnterprise === false);
     const changes = util.filterFormElements(form, true);
 
+    if (changes.month_average_consumption) {
```

Comment:
I am not sure I understand this change.  Why is it necessary?

Reply:
Previously the scope of the form was not include the `month_average_consumption` 
```html 
<input name="month_average_consumption" ...
```.

So for having correct validation style, i include all content of the enterprise page under the form `<form name="EnterpriseForm"`

And this line `const changes = util.filterFormElements(form, true);` i think expose also  
```html 
<input name="month_average_consumption" ... 
``` 
to the change, since we don't need to send that to the enterprise API i just remove it in the submit function, maybe we can do it better

Reply:
Aha, gotcha.